common/sgx/verifier.c

```diff
@@ -302,6 +312,91 @@ static oe_result_t _fill_with_known_claims(
         format_id,
         sizeof(*format_id)));
 
+    // SGX Report CPU SVN
```

Comment:
CPUSVN in SGX `QUOTE`/`REPORT` is a "hint" only. Security analysis should not be made based on the CPUSVN value reported by the SGX enclave SW, including the SGX PCE/QE. The SGX CPU's security property is endorsed by the PCK Cert Chain. The leaf cert of the PCK Cert Chain includes SVNs of CPU firmware/microcode modules that are inside SGX TCB.  Successful `QUOTE` verification, using the PCK Cert Chain, indicates the underlining SGX CPU's security property is equal to or higher than the SVNs in the PCK Cert Chain leaf Cert. The SVNs in the PCK Cert Chain leaf Cert might be associated with a SGX TCB that's out-of-date, ie, with known vulnerability. In that case, it's possible that the enclave SW that produced the SGX `QUOTE` has been compromised and might have spoofed the CPUSVN value in the SGX `QUOTE`. Only if the leaf Cert indicates the SGX CPU is free of any known security vulnerability, should the CPUSVN value in SGX `QUOTE` be trusted.

CPUSVN value in SGX `QUOTE`/`REPORT` is an opaque value. Two different value of CPUSVN can not be numerically compared for TCB level comparison purpose. 

Therefore, exposing the CPUSVN value in SGX `QUOTE` as a claim almost has no value. It should not be included in the outputted claims.

Reply:
Hided CPUSVN from `QUOTE` as we should use CPUSVN from the leaf certificate.

Reply:
Looks good.